src/concuerror_scheduler.erl

```diff
@@ -980,18 +980,22 @@ more_interleavings_for_event([TraceState|Rest], Event, Later, Clock, State,
             end
         end
     end,
-  NC =
-    orddict:store(
-      EarlyActor, EarlyIndex,
-      max_cv(lookup_clock(EarlyActor, EarlyClockMap), Clock)),
-  {NewClock, NewTrace, NewRest} =
+  NewClock =
+    case Action =:= none of
+      true -> Clock;
+      false ->
+        orddict:store(
```

Comment:
Why is the code so ugly here?  How about factoring out the `max_cv(lookup_clock(...)` line and assigning it to a variable before the call to `ordict:..`, which will in turn allow the call to fit in a single line?

Reply:
Yes, indeed, I am cleaning things all over the place (and getting a 4.5x speedup in the meantime). 😄